packages/kbn-search-connectors/types/index.ts

```diff
@@ -7,7 +7,6 @@
  */
 
 export * from './connectors';
-export * from './connectors_api';
```

Comment:
I found the type in `import { Result } from '@elastic/elasticsearch/lib/api/types';` so we don't need to redefine the Result type internally